package.json

```diff
@@ -25,17 +25,21 @@
   "dependencies": {
     "@google-cloud/datastore": "^3.1.0",
     "@google-cloud/storage": "^4.0.0",
-    "compression": "^1.7.4",
```

Comment:
We are removing some middlewares here that we are using:

* Compression is required to gzip the responses since cloud run doesn't do that on it's own. It's not that important here on `api-settings` since responses are already quite small, but we'll need it in `api-events` for example.
* Helmet is a very simple middleware which sets some http headers for additional security. It's not much, but it's a bare minimum that I think we should keep. There's https://github.com/venables/koa-helmet which is the koa version we could use.

Reply:
@andres-arana About the helmet, It's already added koa-helmet.
About the compression, I agree. I think that we should added in all project. I've just added it.